src/ccnl-core/include/ccnl-prefix.h

```diff
@@ -116,11 +116,14 @@ ccnl_prefix_cmp(struct ccnl_prefix_s *pfx, unsigned char *md,
  *
  * @param[in] prefix        Prefix to be compared (shorter of Longest Prefix matching)
  * @param[in] minsuffix     ?
- * @param[in] maxsuffix     ?
+ * @param[in] maxsuffix     ? 
```

Comment:
What is the change on this particular line for?

Reply:
ah .. it's just a revert commit ..

Reply:
I fixed the whitespace back then and this is simply a revert commit, so we puts it back to where it was.